test/powershell/Modules/PSDiagnostics/PSDiagnostics.Tests.ps1

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "PSDiagnostics cmdlets tests" -Tag "CI", "RequireAdminOnWindows" {
+    BeforeAll {
+        $OriginalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if ( -not $IsWindows ) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+    }
+    AfterAll {
+        $Global:PSDefaultParameterValues = $OriginalDefaultParameterValues
+    }
+
+    Context "Test for Enable-PSTrace cmdlet" {
+        it "Should enable Analytic logs for Microsoft-Windows-PowerShell." {
+            Enable-PSTrace -Force
+
+            [XML]$WevtUtilOutput = & wevtutil gl Microsoft-Windows-PowerShell/Analytic /f:xml
+
+            $WevtUtilOutput.Channel.enabled | Should -Be 'true'
+        }
+    }
+
+    Context "Test for Disable-PSTrace cmdlet" {
+        it "Should disable Analytic logs for Microsoft-Windows-PowerShell." {
+            Disable-PSTrace
+
+            [XML]$WevtUtilOutput = & wevtutil gl Microsoft-Windows-PowerShell/Analytic /f:xml
+
+            $WevtUtilOutput.Channel.enabled | Should -Be 'false'
+        }
+    }
+
+    Context "Test for Get-LogProperties cmdlet" {
+        it "Should show properties of Admin logs for 'Microsoft-Windows-PowerShell'." {
+            [XML]$WevtUtilOutput = wevtutil gl Microsoft-Windows-PowerShell/Admin /f:xml
```

Comment:
I'd use name like `$expected`. This makes tests more readable. Below too.

Reply:
Changed